Gemfile

```diff
@@ -83,7 +83,9 @@ gem 'newrelic_rpm'
 gem 'twitter-typeahead-rails'
 gem 'blacklight_range_limit', '~> 7.0'
 gem 'redis', '~> 5.0'
-gem 'geo_monitor', '~> 0.7', github: 'geoblacklight/geo_monitor'
+# Not compatible with GeoBlacklight 4.x
+# https://github.com/geoblacklight/geo_monitor/issues/12
+# gem 'geo_monitor', '~> 0.7', github: 'geoblacklight/geo_monitor'
```

Comment:
What is our plan in general for GeoMonitor once this upgrade is in place? Will we need to work out a way later to use it, or will be table this for good? Thanks

Reply:
I see this down below https://github.com/geoblacklight/geo_monitor/issues/12 so I think that answers that question : )